test/binary/bad-function-body-size.txt

```diff
@@ -0,0 +1,18 @@
+;;; ERROR: 1
+;;; TOOL: run-gen-wasm-interp
+magic
+version
+section("type") { count[1] function params[0] results[1] i32 }
+section("function") { count[1] sig[0] }
+section("code") {
+  0x01 ;; 1 function body
```

Comment:
I like the comments, but you can also use an arbitrary string as a comment: `count[1]` is what I've done in other files